.gitignore

```diff
@@ -1,5 +1,5 @@
 # Rust
-
+Cargo.lock
```

Comment:
The Cargo.lock file must not be ignored. As this is an application project, we want updating dependencies to be a manual approach.

Reply:
Ok, please ignore that … looks like I was taking a look at an older commit and that issue is already resolved.